scripts/pi-hole/js/queries.js

```diff
@@ -136,7 +138,8 @@ $(document).ready(function() {
     });
     $("#all-queries tbody").on( "click", "button", function () {
         var data = tableApi.row( $(this).parents("tr") ).data();
-        if (data[4] === "Pi-holed")
+        status = data[4];
+        if (status.substr(0,2) === "Pi")
```

Comment:
Good catch, will do that, although on the main page it will be much more difficult (it does not know on loading of the page as the blocking reason comes in only later via AJAX).